ext/zip/tests/oo_getexternalattributesname_error.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+ZipArchive::getExternalAttributesName() empty name error
+--DESCRIPTION--
+This test verifies that ZipArchive::getExternalAttributesName() will throw Fatal error: Uncaught ValueError if the first parameter is empty
+--CREDITS--
+Meletios Flevarakis php@flevarakis.xyz
+--EXTENSIONS--
+zip
+--FILE--
+<?php
+
+$zip = new ZipArchive();
+$dirname = __DIR__ . '/';
+
+$zip->open($dirname . "test.zip", ZIPARCHIVE::CREATE);
+
+$a = ZipArchive::OPSYS_DEFAULT;
+$b = ZipArchive::OPSYS_DEFAULT;
+
+$out = $zip->getExternalAttributesName("", $a, $b);
```

Comment:
In order to avoid using `EXPECTF`, you can catch the exception, and print out the message (one example: Zend/tests/constant_arrays.phpt).